Jenkinsfile.integration

```diff
@@ -38,6 +38,18 @@ pipeline {
             }
         }
 
+        stage('Lint') {
```

Comment:
nit: I would say lint playbooks to make it clear it's only linting plays.

Reply:
done